src/challenges/challengeOne/2.Dropdown/Dropdown.test.tsx

```diff
@@ -1,8 +1,33 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Dropdown } from "./Dropdown";
-
-// describe("Dropdown", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { Dropdown, DropdownProps } from "./Dropdown";
+
+describe("Dropdown", () => {
+  const mockOptions = ["Option 1", "Option 2", "Option 3"];
+  const mockLabel = "Dropdown Label";
+  const mockId = "dropdownId";
```

Comment:
You are creating a few abstractions here, which we do to enhance readability, but do they really enhance it in this case? 

We are always aiming for the cleanest approach to writing code, and clean code is code other developers spend less time reading. 

Look at your implementation, then look at this

```
describe("Dropdown", () => {

  const renderComponent = (additionalProps?: Partial<DropdownProps>) => {
    const defaultProps: Partial<DropdownProps> = {
      options: ["Option 1", "Option 2", "Option 3"],
      label: "Dropdown Label",
      id: "dropdownId",
    };
    render(<Dropdown {...defaultProps} {...additionalProps} />);
  };

  ...
```

Are we losing any readability from removing the abstractions? Is it easy to understand the data from the prop name alone? I believe so. 

But we have used three fewer lines of code, to express the same thing. Cleaner.